robocopkaka/bootcamp-29-project
# Move package.json to root folder.

#### What does this PR do?
Move the package.json file to my root folder

#### Description of Task to be completed?
Move the `package.json` file and my `.gitignore` from the server folder to my root folder. Run `npm install` to install all dependencies and create a node_modules folder in the root folder which will be gitignored. Update paths in `.gitignore` file

#### How should this be manually tested?
When folder is cloned, user should be able to run `npm install` from the root folder

#### Any background context you want to provide?
Allows app be deployed to Heroku

#### What are the relevant pivotal tracker stories?(if applicable)
N/A
